Classes/ContentReleaseManager.php

```diff
@@ -72,6 +93,18 @@ public function cancelAllRunningContentReleases()
         }
     }
 
+    public function cancelRunningContentReleases(JobId $jobId): void
```

Comment:
```suggestion
   /** Cancel all running content releases, except a given one */
    public function cancelRunningContentReleases(JobId $exceptJobId): void
```

Reply:
I think I made a mistake here anyway. It doesn't make sense to cancel all except one. Actually the specified one should be canceled 🤦‍♂️